builtin/logical/transit/path_config.go

```diff
@@ -41,27 +41,42 @@ func (b *backend) pathConfigWrite(
 	req *logical.Request, d *framework.FieldData) (*logical.Response, error) {
 	name := d.Get("name").(string)
 
-	// Check if the policy already exists
-	lp, err := b.policies.getPolicy(req, name)
+	// Check if the policy already exists before we lock everything
+	p, lockType, err := b.lm.GetPolicy(req.Storage, name)
```

Comment:
Here is a case where having something like `GetPolicyExclusive` might be useful to avoid the Unlock/Refresh/Unlock cycle